openpype/hosts/fusion/plugins/load/actions.py

```diff
@@ -15,6 +15,7 @@ class FusionSetFrameRangeLoader(load.LoaderPlugin):
                 "pointcache",
                 "render"]
     representations = ["*"]
+    extensions = {"*"}
```

Comment:
This technically is a redundant addition, no? Since it defaults to that value.

Reply:
Yes, it is but  I would rather keep it there for someone to quickly see what it suppose to support, without tracing inheritance.